walkthroughs/week-3-server/examples/random-quotes/pom.xml

```diff
@@ -23,6 +23,11 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
```

Comment:
This file should not be a part of this PR I think.

Reply:
Done.